templates/actor/stats.hbs

```diff
@@ -1,153 +1,151 @@
+          {{formInput
+          systemFields.hero.fields.recoveries.fields.value
+          value=system.hero.recoveries.value
+          }}
+        </div>
+        <span class="resource-divider"> / </span>
+        <span class="resource-max">
           {{#if isPlay}}
-            {{system.hero.recoveries.max}}
+          {{system.hero.recoveries.max}}
           {{else}}
-              {{formInput
-                systemFields.hero.fields.recoveries.fields.max
-                value=system.hero.recoveries.max
-              }}
-          {{/if}}
-          </span>
-        </div>
-      </div>
-      <div class="resource">
-        {{#if system.class}}
-        {{#if isPlay}}
-          <label>
-            <span>{{system.hero.primary.label}}</span>
-            {{formInput
-              systemFields.hero.fields.primary.fields.value
-              value=system.hero.primary.value
-            }}
-          </label>
-          {{#if system.class.system.secondary}}
-          <label>
-            <span>{{system.hero.secondary.label}}</span>
-            {{formInput
-              systemFields.hero.fields.secondary.fields.value
-              value=system.hero.secondary.value
-            }}
-          </label>
-          {{/if}}
-        {{else}}
-        <label class="resource-label">
-          {{localize systemFields.hero.label}}
-        </label>
-        <div class="resource-content">
           {{formInput
-            systemFields.hero.fields.primary.fields.value
-            value=system.hero.primary.value
+          systemFields.hero.fields.recoveries.fields.max
+          value=system.hero.recoveries.max
           }}
-          {{#if system.class.system.secondary}}
-            {{formInput
-              systemFields.hero.fields.secondary.fields.value
-              value=system.hero.secondary.value
-            }}
           {{/if}}
-        </div>
-        {{/if}}
+        </span>
+      </div>
+    </div>
+    <div class="resource">
+      {{#if system.class}}
+      {{#if isPlay}}
+      <label>
+        <span>{{system.hero.primary.label}}</span>
+        {{formInput
+        systemFields.hero.fields.primary.fields.value
+        value=system.hero.primary.value
+        }}
+      </label>
+      {{#if system.class.system.secondary}}
+      <label>
+        <span>{{system.hero.secondary.label}}</span>
+        {{formInput
+        systemFields.hero.fields.secondary.fields.value
+        value=system.hero.secondary.value
+        }}
+      </label>
+      {{/if}}
+      {{else}}
+      <label class="resource-label">
+        {{systemFields.hero.label}}
+      </label>
+      <div class="resource-content">
+        {{formInput
+        systemFields.hero.fields.primary.fields.value
+        value=system.hero.primary.value
+        }}
+        {{#if system.class.system.secondary}}
+        {{formInput
+        systemFields.hero.fields.secondary.fields.value
+        value=system.hero.secondary.value
+        }}
```

Comment:
We might end up disagreeing on what's most readable for a `formInput / formGroup` when it comes to a single line or a line per property as here. I think we should do more data prep so we can just do `{{formInput x.field value=x.value}}`.

Reply:
Maybe we grab that formGroupSimple/formInputSimple from SWADE? Can be a future PR

Reply:
I'm leaning towards having a helper function in the system's scope. One that can take a path, additional data, and create some standard data object that can be fed into the `formGroup` helper directly.